### ISSUE TITLE
Test Search Inside Results Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4905]
# Page to Test:

Search Inside Results - https://openlibrary.org/search/inside?q=The+Crucible&mode=ebooks&has_fulltext=true

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by SaravgiYash** [comment]
@bpmcneilly I would like to work on this issue

**Comment by Yashs911** [comment]
1. When the user tab through both Search Inside and Search Result Page it opens the search-dropdown and it can not be collapsed so my suggestion was as we already have the input in a second search bar we can just remove the input from the top search bar.
2. Also need the focus outline for bookcovers
![image](https://user-images.githubusercontent.com/64412143/114274487-330ae780-9a3c-11eb-90f6-c84334c1008e.png)

**Comment by tuminzee** [comment]
@Yashs911 would you still be interested in working on this issue?

**Comment by mekarpeles** [comment]
Closing — this 2021 manual audit template is superseded by automated a11y tooling now in place: jest-axe CI gate (PR #13006, pending merge) and pa11y site-wide scanning. The Search Inside feature is also a niche path; violations on core pages are the current focus.
